### ISSUE TITLE
rogue talent [Hack and slash] not proccing correctly

### ISSUE THREAD

**Issue by NikaFK** [dalaranwow/dalaran-wow#3617]
**Description**:
Hack and Slash gives the Rogue a chance to deliver a second blow to the same target, while wielding a one-handed sword or axe.
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Now hack and slash proccs from any of the weapons and get a extra swing with that weapon which also includes OH
Aswell as it does not seem to be able to procc of alot of the rogue's abilities.

**Expected behaviour**: Tell us what should happen instead
The extra attack is always done by the weapon in the main hand.
It should also be able to procc from alot rogues abilities
**Steps to reproduce the problem**:

1. have a rogue specced hack and slash using only one sword in MH
2. Equip a fast OH and then look at damage range on proccs
3.  see as it will also procc of the OH which is wrong

**Include proofs for this behaviour**
Using only a MH sword
http://imgur.com/a/Xqmeu
using dual wield swords
http://imgur.com/a/0TRGi

http://wowwiki.wikia.com/wiki/Hack_and_Slash

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

**Comment by robinsch** [comment]
https://en.wikipedia.org/wiki/Law_of_large_numbers

Please only try Sinister Strikes for example. You can turn your character at a certain degree so you can only use special abilitys.

**Comment by Rushor** [comment]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote @http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Does not proc properly for special abilities. I did notice occasional white attacks that would happen after an SS that might have either been random errors with the way I was standing or how hack and slash procs for special abilities on this server (ironically this is what procced the only actual hack and slash attack). Either way this screenshot shot shows that hack and slash does not work properly (if the melee attacks are supposed to be the procs from specials then they are doing significantly less damage)

**Expected behaviour**: Tell us what should happen instead

Hack and Slash should proc off special abilities and do damage equal to the amount that the attack hit for.

**Steps to reproduce the problem**:

1.  Go dummy
2.  Stand at angle so auto attacks wont go off
3.  Spam SS/Evis

![capture](https://cloud.githubusercontent.com/assets/24283210/20778584/88ebf3a2-b73c-11e6-92ba-cdc2f805f02b.PNG)


You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

http://wowwiki.wikia.com/wiki/Hack_and_Slash

**UPDATE**

after more testing and double checking my position by attacking the dummy while in my mutilate spec and making sure i was getting 0 auto attacks it seems that hack and slash procs from SS and evis are being registered as auto attacks and not as hack and slash procs. while this may be better then nothing it is still doing less dmg on average then SS and evis (should be equal) and suffers from white miss chances and glancing blows (also shouldn't happen) reducing overall dps by more then the gain we get from the extra procs (the extra auto attack from the special attacks can themselves proc hack and slash)

screen shots:

![capture3](https://cloud.githubusercontent.com/assets/24283210/20779699/2250228c-b744-11e6-8adf-b9a82fb3cd88.PNG)
![capture4](https://cloud.githubusercontent.com/assets/24283210/20779700/253469b8-b744-11e6-91ec-f2f74c5ef3be.PNG)
![capture2](https://cloud.githubusercontent.com/assets/24283210/20779704/2ad1e120-b744-11e6-922c-9e90416d6d0b.PNG)

Screenshot showing no melee attacks with muti spec from the same positioning:
![capture5](https://cloud.githubusercontent.com/assets/24283210/20779938/d673e91e-b745-11e6-8338-b3c4580a22f3.PNG)

**Comment by InshadowsAlgalon** [comment]
Bump, so whats up with this?

**Comment by tyler200100** [comment]
Why was this closed?

**Comment by Jiaku** [comment]
bump
